pom.xml

```diff
@@ -53,6 +53,21 @@
         </repository>
       </repositories>
     </profile>
+	<profile>
+		<id>javadoc</id>
+		<build>
+			<plugins>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-javadoc-plugin</artifactId>
+					<configuration>
+						<!-- Special code only for J9 -->
+						<excludePackageNames>org.eclipse.osgi.internal.cds</excludePackageNames>
```

Comment:
I think we should exclude *.internal.* as javadoc for internals is not generally required (yet).

Reply:
It is not strictly required, but internal code is still used, so bad javadoc hurts there too, also internal API is widly used across eclipse plugins, so I think its good to have it for now, if it really creates issues, we can later use more explicit excludes.

Reply:
OK. Only thing to consider is that if showing hundreds of warnings very few people (if anyone at all) will try to get a clean build as the battle looks lost but if the number is not that great there are people that try to achieve it. Reiteration with stricter rules can happen after that.

Reply:
I have now enabled some of the options from the isv build, that should reduce warnings/errors. Beside that I hope that at least no **new** ones are introduced that would already be a benefit.